spark/src/test/scala/io/delta/sharing/spark/DeltaSharingFileProfileProviderSuite.scala

```diff
@@ -70,8 +70,8 @@ class DeltaSharingFileProfileProviderSuite extends SparkFunSuite {
     )
   }
 
-  test("version is missing") {
-    val e = intercept[IllegalArgumentException] {
+  test("shareCredentialsVersion is missing") {
+    val e = intercept[Exception] {
```

Comment:
Question: why was this needed? Did the type of error change?

Reply:
The test passed when I changed it back to `IllegalArgumentException` in local, let's see if the PR checks will pass.